CHANGES.md

```diff
@@ -11,6 +11,9 @@ v1.4.0 (unreleased)
 
 * Add support for using degrees in full-sphere projections. [#2279]
 
+* Fixed a bug in when using `compute_statistic` on an array large enough to
+  need chunking when a subset is defined. [#2302]
```

Comment:
```suggestion
* Fixed a bug on setting a return view in `compute_statistic` when a subset
  is defined, resulting in a broadcast error in arrays large enough to need
  chunking. [#2302]
```
I think this captures the full scope of the bug better, even if chunking may be the only case where `view` is used in  practice (maybe that's why it is somewhat treated like a private parameter).